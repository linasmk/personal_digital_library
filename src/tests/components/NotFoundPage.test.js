import React from "react";
import { shallow } from "enzyme";
import NotFoundPage from "../../components/pages/NotFoundPage";

test("Should NotFoundPage render corectly", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
