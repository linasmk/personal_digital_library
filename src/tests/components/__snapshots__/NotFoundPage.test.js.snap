// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should NotFoundPage render corectly 1`] = `
<div>
  <h2>
    404: Page Not Found!
  </h2>
  <Link
    to="/"
  >
    Go back to the main page
  </Link>
</div>
`;
